components/vf-button/vf-button.scss

```diff
@@ -213,3 +219,16 @@ html:not(.vf-disable-deprecated) {
 
 }
 // End deprecated
+
+
+// HACK
```

Comment:
this is what needs to go into the `vf-wp` CSS file -- although there's a potential issue w/ CSS custom properties not crossing CSS stylesheets.

Reply:
Should we comment it out here — that way we don't accidently get used to its effects on non vf-wp sites?

Reply:
if we comment it out here - the border won't work in groups -- because `vf-wp` main.css comes before styles.css … so the custom properties don't cascade.

Unless I'm not understanding your question?

Reply:
Then maybe we can just add the custom properties to the vf-wp workaround CSS file?

Reply:
We could do this - and be sure to update whenever needed. 

It's better to make 'fixes' for platforms using the `vf-core` in the platforms own CSS file at the end of the day.

And when we get to any kind of 'theming' in future `vf-wp` -- perhaps getting `vf-wp` to `npm instal`l the design tokens could be a good thing, moving forward (but not for now)